README.rst

```diff
@@ -1,64 +1,241 @@
+
+The application by default will be listening on port `8080`. Usually
+accessible at `http://127.0.0.1:8080`, it depends on how your Docker is
+configured locally. A Swagger/OpenAPI console will be available at
+`http://127.0.0.1:8080/api/ui/#/default`.
+
+Deploying to AWS
+================
+
+There are many ways to deploy Lizzy to AWS. At Zalando we use `STUPS
+platform`_ that provides a convenient and audit-compliant
+Platform-as-a-Service (PaaS). An example of the `Senza definition`_ to
+deploy Lizzy would be:
+
+.. code-block:: yaml
+
+    SenzaInfo:
+      StackName: lizzy
+      Parameters:
+        - ImageVersion:
+            Description: "Docker image version of lizzy."
+    SenzaComponents:
+      - Configuration:
+          Type: 'Senza::StupsAutoConfiguration'
+      - AppServer:
+          Type: Senza::TaupageAutoScalingGroup
+          AssociatePublicIpAddress: false
+          ElasticLoadBalancer: AppLoadBalancer
+          IamRoles: ['app-lizzy']
+          InstanceType: t2.nano
+          SecurityGroups: ['app-lizzy']
+          TaupageConfig:
+            application_version: '{{Arguments.ImageVersion}}'
+            environment:
+              ALLOWED_USER_PATTERN: "^(jenkins-slave-\\w+)$"
+              DEPLOYER_SCOPE: myscope
+              LANG: C.UTF-8
+              LC_ALL: C.UTF-8
+              LOG_LEVEL: DEBUG
+              REGION: '{{AccountInfo.Region}}'
+              TOKEN_URL: 'https://token.auth.example.com/oauth2/access_token'
+              TOKENINFO_URL: 'https://info.auth.example.com/oauth2/tokeninfo'
+            health_check_path: /api/swagger.json
+            ports: {8080: 8080}
+            runtime: Docker
+            source: 'lizzy:{{Arguments.ImageVersion}}'
+      - AppLoadBalancer:
+          HTTPPort: 8080
+          HealthCheckPath: /api/swagger.json
+          Scheme: internet-facing
+          SecurityGroups: ['app-lizzy-lb']
+          Type: Senza::WeightedDnsElasticLoadBalancer
+
+Access Control for Lizzy
+------------------------
+
+Lizzy application will need to have access to create new Cloud
+Formation stacks and some others services from Amazon API. You will
+need to specify the a `IAM role`_ similar to:
+
+.. code-block:: json
+
+    {
+        "Statement": [
+            {
+                "Action": [
+                    "iam:*",
+                    "cloudformation:*",
+                    "ec2:*",
+                    "route53:*",
+                    "elasticloadbalancing:*",
+                    "cloudwatch:*",
+                    "elasticache:*",
+                    "acm:*",
+                    "autoscaling:*"
+                ],
+                "Effect": "Allow",
+                "Resource": "*"
+            }
+        ],
+        "Version": "2012-10-17"
+    }
+
+That is the minimal configuration needed for Lizzy to be able to run
+successfully Senza commands. Other statements might be included in this
```

Comment:
> Other statements might be included in `this`
Use of `this` here is too vague. 

Also, I don't think it is specified where the above stated json file is saved and by what name.